openpype/settings/defaults/project_settings/maya.json

```
@@ -30,6 +30,36 @@
             }
         ]
     },
+    "RenderSettings": {
+        "apply_render_settings": true,
+        "default_render_image_folder": "",
+        "aov_separator": "underscore",
+        "arnold_renderer": {
+            "image_prefix": "",
```

This needs default sane value.

```suggestion
            "image_prefix": "maya/<Scene>/<RenderLayer>/<RenderLayer>_<RenderPass>",
```